src/app/lib/analyticsUtils/index.js

```diff
@@ -619,3 +619,6 @@ export const getATIMarketingString = (href, campaignType) => {
 };
 
 export const LIBRARY_VERSION = 'simorgh';
+
+export const onOnionTld = () =>
+  onClient() ? window.location.host.endsWith('.onion') : false;
```

Comment:
Ooh never knew endsWith was a method, really nice

Reply:
there is startsWith too 🙂

Reply:
a bit nicer than a regex!